sapi/cgi/tests/bug80849-cgi.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
```

Comment:
Does this really test that the CRLF is present? Do we need to check a second header after it?

Reply:
It checks the whole line; if the CRLF is missing, the next header line is concatenated, and the test fails (without the fix).